### ISSUE TITLE
Add content to sponsors page

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#1314]
We need to add a page that recognises the contributions each sponsor has made to the project - we already have this page, we just need to add content to it :)

We are still waiting on some information for this (see #1238), however, we can certainly still publish this without having all the information at hand.

**Comment by pradyunsg** [comment]
May I give this a shot -- around a couple of weeks from now?

**Comment by nlhkabu** [comment]
I'm not sure that there is anything for you to do here @pradyunsg - we are still in the process of collecting content for this page.

**Comment by pradyunsg** [comment]
I was thinking of leaving the space empty for the remaining sponsors until
they respond and filling in the rest of the details. I don't know if that's
already done though.

**Comment by dstufft** [comment]
I think the main issue here is that this content lives in a private repository, so it's difficult for anyone but myself or Nicole to touch it. It's not part of Warehouse itself.

**Comment by pradyunsg** [comment]
Oh, that's fine then. No issues.

**Comment by di** [comment]
@nlhkabu Did you have any particular plans for this page or should I rough something out?

**Comment by nlhkabu** [comment]
Yes, I've got some ideas - am planning to start on this tomorrow @di.  I will ping you for help when needed :)
